lib/retriable/exponential_backoff.rb

```diff
@@ -28,15 +28,16 @@ def intervals
         [base_interval * multiplier**iteration, max_interval].min
       end
 
-      return intervals if rand_factor.zero?
-
-      intervals.map { |i| randomize(i) }
+      if rand_factor.zero?
```

Comment:
Removing the interval check in randomize makes sense, but in this change, I prefer my guard/return/fall-through style rather than if/else. Can you just revert this change, it looks good otherwise.

Reply:
no problem